domains/*.ultrix.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "*.ultrix",
+
+    "owner": {
+        "email": "pollardjackson@outlook.com"
+    },
+
+    "record": {
+        "CNAME": "not-setup-yet"
```

Comment:
Invalid CNAME record, domain reservation is not allowed.

Reply:
can i jus delete the file it don't want it anymore.

Reply:
i deleted the file